backend/kernelCI_app/views/issueDetailsView.py

```diff
@@ -0,0 +1,56 @@
+from typing import Dict, Optional
+from django.http import HttpResponseBadRequest, HttpResponseNotFound, JsonResponse
+from django.views import View
+from kernelCI_app.models import Issues
+from kernelCI_app.utils import getErrorResponseBody
+
+
+class IssueDetails(View):
+    def _fetch_issue(self, *, issue_id: str, version: int) -> Optional[Dict]:
+        issue_fields = [
+            "field_timestamp",
+            "id",
+            "version",
+            "origin",
+            "report_url",
+            "report_subject",
+            "culprit_code",
+            "culprit_tool",
+            "culprit_harness",
+            "build_valid",
+            "test_status",
+            "comment",
+            "misc",
+        ]
+
+        query = Issues.objects.values(*issue_fields).filter(
+            id=issue_id, version=version
+        )
+        if len(query) == 0:
+            return None
+        return query[0]
+
+    def get(self, _request, issue_id: str | None, version: str | None):
```

Comment:
I believe `JsonResponse` is a valid return type here. Assuming this (https://github.com/kernelci/dashboard/pull/689/files#diff-c22db5b2e8c77ae1df9a53af32a177abc1ef9547568d5a9ff8e47e7cb5b9357c) is correct